### PR TITLE
Add simple example working on Rx72n

### PR DESCRIPTION
**Pull request by miyazakh**
Use T4-Tiny IP stack

# Description

Simple use cases including crypt, benchmark, client and server by using T4-Tiny Renesas IP stack.

# Testing

Run programs on RX72N

# Checklist

 - [ ] added tests
 - [ ] updated/added doxygen
 - [x] updated appropriate READMEs
 - [ ] Updated manual and documentation

### CODE REVIEW

File: IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/client/simple_tls_tsip_client.c [hunk 1]
```diff
@@ -0,0 +1,293 @@
+/* wolf_client.c
```
Comment (cconlon) [inline IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/client/simple_tls_tsip_client.c:1]: File name should be "simple_tls_tsip_client.c"

File: IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/server/simple_tcp_server.c [hunk 2]
```diff
@@ -0,0 +1,92 @@
+/* wolf_server.c
```
Comment (cconlon) [inline IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/server/simple_tcp_server.c:1]: File name should be "simple_tcp_server.c"

File: IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/server/simple_tls_server.c [hunk 3]
```diff
@@ -0,0 +1,194 @@
+/* wolf_server.c
```
Comment (cconlon) [inline IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/server/simple_tls_server.c:1]: File name should be "simple_tls_server.c"

File: IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/wolfssl_simple_demo.h [hunk 4]
```diff
@@ -0,0 +1,62 @@
+/* wolfssl_demo.h
```
Comment (cconlon) [inline IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/wolfssl_simple_demo.h:1]: File name should be "wolfssl_simple_demo.h"

**Review by cconlon — CHANGES_REQUESTED** [review]
Just a few minor comment corrections.

File: IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/server/simple_tcp_server.c (hunk 2 above)
Comment (miyazakh) [reply IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/server/simple_tcp_server.c:1]: Thanks. Fixed

File: IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/server/simple_tls_server.c (hunk 3 above)
Comment (miyazakh) [reply IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/server/simple_tls_server.c:1]: Thanks. Fixed

File: IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/wolfssl_simple_demo.h (hunk 4 above)
Comment (miyazakh) [reply IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/wolfssl_simple_demo.h:1]: Thanks. Fixed

**Comment by miyazakh** [comment]
Will push changes once code freeze is off.

**Review by cconlon — APPROVED** [review]
Approved, will merge when code freeze has been lifted.

**Comment by miyazakh** [comment]
@cconlon 
Code freeze is off now. Could you merge this?

**Pull request merged**